### ISSUE TITLE
Cannot read property 'sendWithPromise' of null

### ISSUE THREAD

**Issue by esetnik** [react-pdf-viewer/react-pdf-viewer#383]
When loading a second fileUrl into the `Viewer` it throws an error.

```
Uncaught TypeError: Cannot read property 'sendWithPromise' of null
    at WorkerTransport.getPage (api.js:2499)
    at PDFDocumentProxy.getPage (api.js:664)
    at core.js:1710
    at invokePassiveEffectCreate (react-dom.development.js:23487)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994)
    at invokeGuardedCallback (react-dom.development.js:4056)
    at flushPassiveEffectsImpl (react-dom.development.js:23574)
    at unstable_runWithPriority (scheduler.development.js:646)
    at runWithPriority$1 (react-dom.development.js:11276)
    at flushPassiveEffects (react-dom.development.js:23447)
    at react-dom.development.js:23324
    at workLoop (scheduler.development.js:590)
    at flushWork (scheduler.development.js:545)
    at MessagePort.performWorkUntilDeadline (scheduler.development.js:157)
```

```
The above error occurred in the <PageSizeCalculator> component:

    at PageSizeCalculator (https://project.com/static/js/0.chunk.js:45953:16)
    at DocumentLoader (https://project.com/static/js/0.chunk.js:46133:26)
    at LocalizationProvider (https://project.com/static/js/0.chunk.js:43472:21)
    at ThemeProvider (https://project.com/static/js/0.chunk.js:46249:21)
    at Viewer (https://project.com/static/js/0.chunk.js:46259:15)
    at div
    at Reading__ViewerContainer (https://project.com/static/js/0.chunk.js:317535:6)
    at div
```

Appears to be related to https://github.com/VadimDez/ng2-pdf-viewer/issues/382

**Comment by esetnik** [comment]
Note: I reverted my project to the old `@phuocng/react-pdf-viewer@1.7.0` and I don't experience the issue.

**Comment by phuoc-ng** [comment]
@esetnik Can you let me know how to reproduce the issue?

I tried to change the `fileUrl` with a simple script as

```jsx
import React, { useState } from 'react';
import { PrimaryButton, Viewer, Worker } from '@react-pdf-viewer/core';
import { defaultLayoutPlugin } from '@react-pdf-viewer/default-layout';

import '@react-pdf-viewer/default-layout/lib/styles/index.css';

// Test component
const files = [
    'http://localhost:8001/pdf-open-parameters.pdf',
    'http://localhost:8001/pdf-sample.pdf',
];

const [fileUrl, setFileUrl] = useState(files[0]);
const defaultLayoutPluginInstance = defaultLayoutPlugin();

return (
    <Worker workerUrl="https://unpkg.com/pdfjs-dist@2.5.207/build/pdf.worker.js">
        <div style={{ marginBottom: '16px' }}>
            <PrimaryButton onClick={() => setFileUrl(files[1])}>Change file</PrimaryButton>
        </div>

        <div
            style={{
                height: '750px'
            }}
        >
            <Viewer
                fileUrl={fileUrl}
                plugins={[
                    defaultLayoutPluginInstance,
                ]}
            />
        </div>
    </Worker>
);
```

Clicking the `Change file` button will load the second PDF document. It works as you can see the following gif

![change-file-url](https://user-images.githubusercontent.com/57786711/101500560-37959580-39a1-11eb-89b6-681d975fbc4b.gif)

**Comment by manicmonkey** [comment]
@phuoc-ng this is reproducible using the preview example: https://react-pdf-viewer.dev/examples/preview-a-pdf-file-before-uploading/

Just browse to a file, and then browse to another file.

**Comment by phuoc-ng** [comment]
@manicmonkey It was fixed and will be included in the next 2.3.2
See the issue here #417

**Comment by phuoc-ng** [comment]
@manicmonkey @esetnik 
Please upgrade to [v2.3.2](https://react-pdf-viewer.dev/updates) which fixes the issue.
